src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,17 @@
-export const UserInfo = () => {};
+import React from 'react';
+
+import './UserInfo.scss';
+
+import { User } from '../../types';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return (
+    <a className="UserInfo" href={`mailto:${user?.email}`}>
+      {user?.name}
+    </a>
```

Comment:
[REACT KNOWLEDGE] - Don't render the component if the property that you pass to the component has `null` or `undefined` value. In this case, you should check if the user is defined before rendering the `UserInfo` component. This would typically be handled in the parent component that renders `UserInfo`, ensuring that `UserInfo` is only rendered when there is a valid user object.

Reply:
good point